infra/modules/waf/main.tf

```
@@ -0,0 +1,87 @@
+resource "aws_wafv2_web_acl" "waf" {
+  # checkov:skip=CKV2_AWS_31: Need to ask team about using Firehose
+  name        = "wic-prp-waf"
+  description = "Managed ruleset for the WIC PRP project."
```

Let's make this module more generic:
```suggestion
  name        = var.waf_name
  description = "Managed ruleset for WAF"
```
You'll need to add a variables.tf file. I think we want to call this module in env-template, yes?